corehq/apps/cloudcare/static/cloudcare/js/formplayer/apps/models.js

```diff
@@ -0,0 +1,9 @@
+/*global FormplayerFrontend */
+
+FormplayerFrontend.module("Apps.Models", function (Models, FormplayerFrontend, Backbone) {
+    Models.App = Backbone.Model.extend({
+        urlRoot: "appSelects",
```

Comment:
i copied this, but i don't actually think it does anything. didn't want to make code changes on this pass